bin/mint-release-branch

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env perl
+# PODNAME: mint-release-branch
+# ABSTRACT: build a release branch from labeled merge requests
+
+use v5.20;
+use lib 'lib';
+use App::MintTag;
+
+binmode *STDOUT, ':encoding(UTF-8)';
+binmode *STDERR, ':encoding(UTF-8)';
+
+use App::MintTag::Logger '$Logger' => { init => {
+  ident     => 'mint-tag',
+  facility  => 'daemon',
+  to_stdout => 1,
+  log_pid   => 0,
+} };
+
+use Getopt::Long::Descriptive;
+
+my ($opt, $usage) = describe_options(
+  '%c %o',
+  [ 'config|c=s', 'config file to use', { required => 1 } ],
+  [ 'auto',       'do not run in interactive mode' ],
+  [ 'verbose|v',  'turn on debug logging' ],
+  [ 'repo|r=s',   'repo to use' ],
+  [ 'mr|pr=s@',   'if given, only allow these merge request numbers' ],
```

Comment:
I wonder if you want a canonical name of merge_requests so that later $foo->mr->@* is never confusing.  Maybe the context here is small enough.

(This makes me think a "make the accessor for this called X would be useful sometimes".)

Reply:
Yeah, I think this program is short enough that I think having `$opt->mrs` is clear enough; in the program proper, it's always `$extra_mrs`.